scripts/create_baseline_stubs.py

```diff
@@ -152,22 +152,38 @@ def add_pyright_exclusion(stub_dir: str) -> None:
     assert i < len(lines), f"Error parsing {PYRIGHT_CONFIG}"
     while not lines[i].strip().startswith("]"):
         i += 1
-    # Must use forward slash in the .json file
-    line_to_add = f'        "{stub_dir}",'.replace("\\", "/")
-    initial = i - 1
-    while lines[i].lower() > line_to_add.lower():
+    end = i
+
+    # We assume that all third-party excludes must be at the end of the list.
+    # This helps with skipping special entries, such as "stubs/**/@tests/test_cases".
```

Comment:
Ideally we'd assert this somehow in `check_consistent.py`, but I can't think of a simple way of doing so, and a complex way of doing so would probably be overkill for this use case. If this assumption becomes invalidated at some point, we can probably just deal with that when the problem arises.